src/components/table/table-row/table-row.style.js

```diff
@@ -122,4 +124,13 @@ function dragRowStyling({ isDragged, isDragging, theme }) {
     `}
   `;
 }
+
+StyledTableRow.propTypes = {
+  theme: PropTypes.object
+};
+
+StyledTableRow.defaultProps = {
+  theme: baseTheme
```

Comment:
Interesting – so moving `.defaultProps = { theme: baseTheme }` from `table-row.component.js` to `table-row.style.js` resolves the issue of the Classic theme being lost on `<Table>` ?

Maybe this is something we need to document – that `.defaultProps = { theme: baseTheme }` must always be added to the `*.style.js` file (not the `*.component.js` file) ?

Reply:
Yeah the default theme is only needed at the component level when we need to use the `withTheme` hoc for some conditional functionality, surprised it broke it though